upgrade/1.5.2/CSM-With-Other-Products.md

```diff
@@ -0,0 +1,279 @@
+# CSM Only Upgrade on a System with Other Products
+
+This page provides guidance for systems with products installed that are performing an upgrade from a CSM `v1.5.X` release to the CSM `v1.5.2` release.
+
+The [`v1.5.2` upgrade page](../1.5.2/README.md) will refer to this page
+during [Update NCN images](../1.5.2/README.md#update-ncn-images).
+
+**If exclusively CSM is installed on the system, return to [Update NCN images](../1.5.2/README.md#update-ncn-images) and
+choose option 1.**
```

Comment:
Should this be "option 2"?
![image](https://github.com/Cray-HPE/docs-csm/assets/7772179/8182d5fc-4e20-4587-9c38-610d87eaad25)

Reply:
Yes, it should be! I have fixed this.